src/app/components/client/ComboBox.module.scss

```diff
@@ -1,70 +1,5 @@
 @import "../../tokens";
 
-.comboBox {
-  background: $color-white;
-  display: flex;
-  flex-direction: column;
-  position: relative;
-
-  .inputField {
-    border: 1px solid $color-grey-30;
-    border-radius: $border-radius-sm;
-    color: $color-black;
-    // Add a bit more vertical space: The next spacing step is too much.
-    padding: calc($spacing-sm * 1.5) $spacing-md;
-
-    &::placeholder,
-    &.noValue {
-      color: $color-grey-40;
-    }
-
-    &.hasError {
-      border-color: $color-error;
-
-      &:focus {
-        border: 1px solid $color-error;
-        outline: $border-focus-width solid $color-error-focus;
-      }
-    }
-
-    &:focus {
-      border: 1px solid $color-informational-active;
-      outline: $border-focus-width solid $color-informational-focus;
-    }
-  }
-
-  .inputLabel {
-    font-weight: 600;
-    margin-bottom: $spacing-sm;
-  }
-
-  .inputMessage {
-    background: inherit;
-    bottom: 0;
-    color: $color-error;
-    font: $text-body-xs;
-    left: 0;
-    margin: 0 $spacing-sm;
-    padding: 0 $spacing-sm;
-    position: absolute;
-    // the negative y-offset is 50% of the line-height plus a bit extra for
-    // visual balance in relation to the input border endings left and right
-    transform: translateY(calc(100% - 0.85em));
-  }
-
-  .list {
-    list-style: none;
-    margin: 0;
-    padding: 0;
-    position: absolute;
-    width: 100%;
-  }
-
-  .item {
-    padding: 0;
-  }
-}
-
```

Comment:
Thanks for calling that out, makes reviewing a lot easier.

I'm generally wary of re-using CSS modules across different components (that's the opposite of what they're intended for :grin:), since the reason they're different components might very well also be a reason for their styles to diverge in the future, but I suppose the combo box and input field are pretty likely to be styled the same in the future as well, given their similar interaction models.